src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -368,68 +368,101 @@ void getHearingPriority(String isAdjournCase, String isUrgentCase, String expect
-        given(referenceDataServiceHolder.getVenueService()).willReturn(venueService);
+        checkHearingLocationResults(HearingsDetailsMapping.getHearingLocations(caseData, referenceDataServiceHolder),
+                                    EPIMS_ID_1, EPIMS_ID_2);
+    }
 
-        List<HearingLocation> result = HearingsDetailsMapping.getHearingLocations(caseData, referenceDataServiceHolder);
+    @DisplayName("When a case has been adjourned and a different venue has been selected, return the new venue")
+    @Test
+    void getHearingLocationsAdjournmentNewVenue() {
+        buildOverrideHearingLocations();
+
+        setupAdjournedHearingVenue(SOMEWHERE_ELSE.getValue());
+
+        given(referenceDataServiceHolder.isAdjournmentFlagEnabled()).willReturn(false);
 
-        assertThat(result)
-            .hasSize(1)
-            .extracting("locationId","locationType")
-            .containsExactlyInAnyOrder(tuple("219164", COURT));
+        checkHearingLocationResults(HearingsDetailsMapping.getHearingLocations(caseData, referenceDataServiceHolder),
+                                    EPIMS_ID_1);
     }
 
-    @DisplayName("When override Hearing Venue Epims Ids is not empty getHearingLocations returns the override values")
+    @DisplayName("When a case has been adjourned and the same venue has been selected, return the same venue")
     @Test
-    void getHearingLocationsOverride() {
+    void getHearingLocationsAdjournmentSameVenue() {
+        buildOverrideHearingLocations();
 
-        SscsCaseData caseData = SscsCaseData.builder()
-            .appeal(Appeal.builder()
-                .hearingOptions(HearingOptions.builder().build())
-                .build())
-            .schedulingAndListingFields(SchedulingAndListingFields.builder()
-                .overrideFields(OverrideFields.builder()
-                    .hearingVenueEpimsIds(List.of(
-                        CcdValue.<CcdValue<String>>builder()
-                            .value(CcdValue.<String>builder()
-                                .value("219164")
-                                .build())
-                            .build(),
-                        CcdValue.<CcdValue<String>>builder()
-                            .value(CcdValue.<String>builder()
-                                .value("436578")
-                                .build())
-                            .build()))
-                    .build())
-                .build())
-            .processingVenue(PROCESSING_VENUE_1)
-            .build();
+        setupAdjournedHearingVenue(SAME_VENUE.getValue());
+
+        caseData.setHearings(Collections.singletonList(Hearing.builder()
+                    .value(uk.gov.hmcts.reform.sscs.ccd.domain.HearingDetails.builder()
+                    .venueId(EPIMS_ID_2).build())
+                .build()));
+
+        given(referenceDataServiceHolder.isAdjournmentFlagEnabled()).willReturn(false);
```

Comment:
This should be willReturn(true) for the feature to be enabled, and therefore your code to run

Reply:
Done (moved the mock call to buildOverrideHearingLocations() as that covers all the tests that need it

Reply:
ok